articles/vmware/vmw-how-access-centos-update-servers.md

```diff
@@ -31,55 +31,105 @@ UKCloud's Assured OFFICIAL security domain is internet facing, so you need to co
-    ```
-
-Or use the following command to download the repo file directly from the server:
-
-    ```
-    wget -P /etc/yum.repos.d http://<elevated-public-ip-address>/repos/ukcloud_centos6.repo
-    ```
-
-In the above string, replace `elevated-public-ip-address` with the Elevated OFFICIAL public IP addresses. If you're not sure what this is, contact UKCloud Support.
-
-Remove all other `*.repo` files in this directory.
+We provide repositories and EPEL (Extra Packages for Enterprise Linux) for CentOS 6 and 7 on our Elevated OFFICIAL security domain.
+
+To access CentOS base files, create a file called `centos_<x>_il3.repo` in `/etc/yum.repos.d`, where `<x>` is either 6 or 7 depending on your version of CentOS, and populate it with the following:
+
+**CentOS 6:**
+
+```none
+[base]
+name=CentOS-6-Base
+baseurl=http://<elevated-public-ip-address>/centos-base/
+gpgcheck=1
+gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-6
+
+#released updates
+[updates]
+name=CentOS-6-Updates
+baseurl=http://<elevated-public-ip-address>/centos-updates/
+gpgcheck=1
+gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-6
+
+#additional packages that may be useful
+[extras]
+name=CentOS-6-Extras
+baseurl=http://<elevated-public-ip-address>/centos-extras/
+gpgcheck=1
+gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-6
+```
+
+**CentOS 7:**
+
+```none
+[base]
+name=CentOS-7-Base
+baseurl=http://<elevated-public-ip-address>/centos-7-base/
+gpgcheck=1
+gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7
+
+#released updates
+[updates]
+name=CentOS-7-Updates
+baseurl=http://<elevated-public-ip-address>/centos-7-updates/
+gpgcheck=1
+gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7
+
+#additional packages that may be useful
+[extras]
+name=CentOS-7-Extras
+baseurl=http://<elevated-public-ip-address>/centos-7-extras/
+gpgcheck=1
+gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7
+```
+
+Where `<elevated-public-ip-address>` is the Elevated OFFICIAL public IP address (if you're not sure what this is, contact UKCloud Support).
+
+To access CentOS EPEL files, create a file called `epel_<x>_il3.repo` in `/etc/yum.repos.d`, where `<x>` is either 6 or 7 depending on your version of CentOS, and populate it with the following:
+
+**CentOS 6:**
+
+```none
+[epel]
+name=Extra Packages for Enterprise Linux 6
+baseurl=http://<elevated-public-ip-address>/epel/
+enabled=1
+metadata_expire=7d
+gpgcheck=1
+gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-EPEL-6
+```
+
+**CentOS 7:**
+
+```none
+[epel]
+name=Extra Packages for Enterprise Linux 7
+baseurl=http://<elevated-public-ip-address>/epel-7/
+enabled=1
+metadata_expire=7d
+gpgcheck=1
+gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-EPEL-7
+```
+
+To download the repo file directly from the server, use the following command (`<x>` is either 6 or 7 depending on your version of CentOS):
+
+**CentOS base updates and extras:**
+
+```none
+wget -P /etc/yum.repos.d/ http:///<customer_repo_files>/centos_<x>_il3.repo
```

Comment:
Too many /// and missing elevated IP
The URLs here should be 

http://<elevated-public-ip-address>/customer_repo_files/centos_<x>_il3.repo
and
http://<elevated-public-ip-address>/customer_repo_files/epel_<x>_il3.repo